src/main/java/org/kiwiproject/registry/model/ServiceInstance.java

```diff
@@ -38,6 +41,7 @@
     private final String commitRef;
     private final String description;
     private final String version;
+    private final Instant upSince;
```

Comment:
Do we ever return `ServiceInstance` objects where any of the other fields can be `null`? Because we don't really document them, and I would not expect an `upSince` to ever permit a `null` value...

Reply:
ok, what do you want me to do ?

Reply:
See above...my vote is to default it to zero (`Instant.EPOCH`)...that will quickly get someone's attention if they are not setting the `upSince`